packages/mobile/src/firebase/saga.ts

```diff
@@ -153,6 +183,25 @@ export function* watchPaymentRequestStatusUpdates() {
   yield takeLeading(Actions.PAYMENT_REQUEST_UPDATE_STATUS, updatePaymentRequestStatus)
 }
 
+function* updatePaymentRequestNotified({ id, notified }: UpdatePaymentRequestNotifiedAction) {
+  try {
+    Logger.debug(TAG, 'Updating payment request', id, `notified: ${notified}`)
+    yield call(() =>
+      firebase
+        .database()
+        .ref(`${REQUEST_DB}/${id}`)
+        .update({ notified })
+    )
+    Logger.debug(TAG, 'Payment request notified updated', id)
+  } catch (error) {
+    Logger.error(TAG, `Error while updating payment request ${id} status`, error)
```

Comment:
we should probably show the user an error here. Please add a new error message and call `yield put showError` here

Reply:
Ditto for above in updatePaymentRequestStatus